bigquery_etl/cli/metadata.py

```diff
@@ -61,22 +61,44 @@ def metadata(ctx):
 @click.argument("name")
 @project_id_option()
 @sql_dir_option
-def update(name: str, sql_dir: Optional[str], project_id: Optional[str]) -> None:
+@parallelism_option()
+def update(
+    name: str, sql_dir: Optional[str], project_id: Optional[str], parallelism: int
+) -> None:
     """Update metadata yaml file."""
-    table_metadata_files = paths_matching_name_pattern(
-        name, sql_dir, project_id=project_id, files=["metadata.yaml"]
+    table_metadata_files = list(
+        set(
```

Comment:
Is it possible for paths_matching_name_pattern to return duplicates?  It would be a bit cleaner to put this logic in the function instead of each caller